features/features.js

```diff
@@ -0,0 +1,11 @@
+const { initFeatureFlags } = require('@near-wallet/feature-flags');
```

Comment:
Note that this resolves to installing the actual NPM package in production builds, but links to the local filesystem version of the package during local dev so you can change `@near-wallet/feature-flags` and see your changes immediately take effect in e.g. `frontend`